src/XRP/issued-currency-client.ts

```diff
@@ -1222,13 +1223,30 @@ export default class IssuedCurrencyClient {
    *               It is not considered an error if the offer specified does not exist.
    * @param expiration (Optional) Time after which the offer is no longer active, in seconds since the Ripple Epoch.
    *               (See https://xrpl.org/basic-data-types.html#specifying-time)
+   * @param passive (Optional) If enabled, the offer does not consume offers that exactly match it, and instead becomes
```

Comment:
I wonder if we should specify "Defaults to false." for all of these opt flag params?